src/main/java/utils/Waits.java

```diff
@@ -0,0 +1,12 @@
+package utils;
+
+public class Waits {
+
+    public static void waitSeconds(int seconds) {
+        try {
+            Thread.sleep(seconds * 1000);
```

Comment:
We shouldn't have Thread sleep in our frame work, please replace it with ExpectedConditions

Reply:
Updated